### ISSUE TITLE
Text color of disabled input fields on iOS/Safari are not readable

### ISSUE THREAD

**Issue by cat-mandres** [vmware/clarity#5522]
## Describe the bug

Disabled input fields are very difficult to read on iOS/Safari on the default grayish background color.
In modals they are not readable at all because the text color is the same as the background color.

Setting the color to e.g. black fixes this problem.

## How to reproduce

https://clarity-v4-light-theme-2zvj5m.stackblitz.io

Open the stackblitz once in Chrome (everything visible there)
Open the stackblitz once in Safari (the input field cant be read

## Expected behavior

The value in the input field should be easily readable

## Versions

**App**

See stackblitz

**Device:**

- Type: MacBook & iPad
- OS: iOS 14
- Browser: Safari
- Version: 14

## Additional notes

The problem seems to be that the background property of `.clr-input:not([readonly])` is interpreted differently by the 2 browers.

I could not get a stackblitz running with clarity version 5.0.0 because there was no template and the version upgrade failed.

So it is using version 4.0.0 but my application where the error was intially found is using clarity 4.0.9. So if this has actually been fixed in the upgrade to 5.0.0 please let me know

**Comment by Jinnie** [comment]
CC @scroniser Santina, could we please verify if we're tracking this in our internal tools?

**Comment by scroniser** [comment]
@cat-mandres - thank you for reporting this for consideration. Inactive text, such as a disabled input, is considered exempt under [WCAG 1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum). It's possible that this requirement will change in WCAG 3.0.

@Jinnie -- FYI, no change needed under current a11y guidance.

**Comment by Jinnie** [comment]
Thank you, @scroniser 
I'll add this to our Support DB, so we can easily answer future questions in this direction.
